test/verify/check-realms

```diff
@@ -259,6 +258,16 @@ class TestKerberos(MachineCase):
         self.configure_kerberos()
         self.machine.restart_cockpit()
 
+
+        # These only support the default keytab location
+        keytab = "/etc/cockpit/krb5.keytab"
+        if self.machine.image in ["rhel-7-4", "rhel-7-5"]:
```

Comment:
ATM the rhel-7-5 image installs all built rpms, at least until it gets released; so it actually ought to work there. But I suppose disabling it there already is okay, in case we really don't rebase again (and then we can still adjust the test).

Reply:
I did it like this because the goal of the 7.5 image is to test what we deliver there. Seems unlikely that this change will make it there.